### ISSUE TITLE
[BUG] "Hide Recommended Channels" not working properly

### ISSUE THREAD

**Issue by Squtzy** [SevenTV/Extension#931]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest nightly version

- [X] I am using the latest nightly version

### What browsers are you seeing the problem on?

Firefox

### Current Behavior

When using the setting "Hide Recommended Channels" it does not remove the recommended channels in sidebar, instead it removes "X Viewers Also Watch" making the setting "Hide Viewers Also Watch" not do anything at all. 

Issue persists over multiple PC's, with firefox using custom & default settings. Reinstalling 7TV extension, with default and custom settings. So when using both these settings the recommended channels is still in the sidebar. 

TL:DR
Hide Recommended Channels, removes "Viewers Also Watch".
Hide Viewers Also Watch, setting does nothing because of previous setting removing the wrong part of sidebar.

### Expected Behavior

I expected the "Hide Recommended Channels" setting to remove the "Recommended Channels" section of my sidebar, not the "viewers also watch". 
Then I expected the "Hide Viewers Also Watch" setting to remove that section of the sidebar but instead this does not do anything due to the recommended channels bug.

### Steps To Reproduce

Installing/using 7TV or 7TV Nightly, turn on "Hide Recommended Channels" - This removes wrong section. Turning on "Hide Viewers also watch" setting does nothing at the moment.

**Comment by Squtzy** [comment]
bump

**Comment by Squtzy** [comment]
bump

**Comment by junsupark94** [comment]
I checked on my Mac M2 Sonoma 14.0 latest Firefox. It behaves correctly for me. Is it still an issue for you?

**Comment by Squtzy** [comment]
> I checked on my Mac M2 Sonoma 14.0 latest Firefox. It behaves correctly for me. Is it still an issue for you?

Yep it sadly is still doing this. Tried it out on Chrome quickly and same thing there. Idk why it's doing this :/

**Comment by junsupark94** [comment]
What OS are you on? Have you tried incognito mode for both Chrome and Firefox with only 7TV enabled?

**Comment by Squtzy** [comment]
> What OS are you on? Have you tried incognito mode for both Chrome and Firefox with only 7TV enabled?

Windows 11, tried incognito on both browsers.

**Comment by pimothyxd** [comment]
It seems like this is occurring because it currently expects the recommended section to be in a specific place.
https://github.com/SevenTV/Extension/blob/a44aa6e446fa9979fa4479dd1e297cedff07926b/src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue#L207-L211
This should however be fixed in the following pull request #767

**Comment by Squtzy** [comment]
> It seems like this is occurring because it currently expects the recommended section to be in a specific place.
> 
> https://github.com/SevenTV/Extension/blob/a44aa6e446fa9979fa4479dd1e297cedff07926b/src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue#L207-L211
> 
> This should however be fixed in the following pull request #767

Is the extension not being actively updated anymore? Been a long time since I last got a update for it. Hopefully this is part of the next update.

**Comment by pimothyxd** [comment]
> > It seems like this is occurring because it currently expects the recommended section to be in a specific place.
> > https://github.com/SevenTV/Extension/blob/a44aa6e446fa9979fa4479dd1e297cedff07926b/src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue#L207-L211
> > 
> > This should however be fixed in the following pull request #767
> 
> Is the extension not being actively updated anymore? Been a long time since I last got a update for it. Hopefully this is part of the next update.

Good question. I do think the extension is still actively being updated, however it seems to have slowed down a bit compared to a few months back.

**Comment by HandsomeStrife** [comment]
This is also happening for me, same issue

**Comment by ftk789** [comment]
This has been fixed. closing issue
